client/redux/index.js

```diff
@@ -0,0 +1,14 @@
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import axios from 'axios';
+import thunkMiddleware from 'redux-thunk';
+
+const appReducer = combineReducers({});
```

Comment:
The page is immediately crashing with this error: Store does not have a valid reducer. Make sure the argument passed to combineReducers is an object whose values are reducers.

I think this line is causing it. Maybe we should hold off on bringing in redux until we know what state we want the client to be tracking

Reply:
Agreed. I removed redux for now. We can add it back in later if needed.